scss/main.scss

```diff
@@ -46,6 +46,12 @@ $animation-length: 80s;
   pointer-events: none !important;
 }
 
+@media only screen and (min-width: 1750px) {
+    .intro{
```

Comment:
a couple things!
1. i think you forgot to run `npm run web` after making this change :(
2. css definitions are cascading which means that because this is written before the other place where we [define .intro's margin-top](https://github.com/gotechnica/technica-site-2020/blob/master/scss/main.scss#L469), it'll override this media query
![image](https://user-images.githubusercontent.com/41929993/90849807-26ffd780-e33e-11ea-8e72-0225b9475643.png)
3. you can test out wider screen widths by choosing the "responsive" option in chrome inspect and inputting any custom screen width - right now the margin-top: 5% would look something like this for a wide screen 😢 
![image](https://user-images.githubusercontent.com/41929993/90850049-b73e1c80-e33e-11ea-912b-040d7699e41f.png)
![image](https://user-images.githubusercontent.com/41929993/90850107-d3da5480-e33e-11ea-97a3-61d5476ffcb8.png)

Reply:
i thought i ran it but i guess not 😥  i wasn't thorough enough in testing so thanks for the reminders!!